vars/longevityPipeline.groovy

```diff
@@ -192,13 +192,11 @@ def call(Map pipelineParams) {
             }
             stage('Create SCT Runner') {
                 steps {
-                    catchError(stageResult: 'FAILURE') {
-                        script {
-                            wrap([$class: 'BuildUser']) {
-                                dir('scylla-cluster-tests') {
-                                    timeout(time: 5, unit: 'MINUTES') {
-                                        createSctRunner(params, runnerTimeout , builder.region)
-                                    }
+                    script {
```

Comment:
Doing so it will try to run all other stages.
I think we should do the same actions in ccase of error like in the `Preparation` stage:

```
                            currentBuild.result = 'FAILURE'
                            error('SCT runner creation failed')
```

Reply:
run job show it does not run the next steps

Reply:
> Doing so it will try to run all other stages. I think we should do the same actions in case of error like in the `Preparation` stage:
> 
> ```
> currentBuild.result = 'FAILURE'
> error('SCT runner creation failed')
> ```

But won't `catchError` will catch it and move forward?
Current solution just stops the pipeline (excluding 'Post Actions` step which does always some cleanup and sends email anyway)

Reply:
From another side, yes, doesn't really make any difference. Fast test end still is the result.